### DISCUSSION TITLE
Como bloquear um repositório alheio?

### DISCUSSION THREAD

**Discussion by suissa** [frontendbr/forum#668] [Discussão]
Gostaria de saber se alguém aqui sabe como se bloqueia um repositório de outrém como os haters do Be MEAN fizeram com esse meu repositório https://github.com/suissa/Curso-JavaScript-Super-Sayajin ??

Ainda bem que já criei um novo e criarei quantos forem necessários mas o conteúdo não será bloqueado graças à meia dúzia de haters.

https://github.com/suissa/Curso-JavaScript-Super-Sayajin-oficial

Ainda quero saber como fizeram isso.

**Comment by TiagoDanin** [comment]
Parece que quebra dois termos de uso do Github
https://help.github.com/articles/github-terms-of-service/#2-github-repositories
https://help.github.com/articles/github-terms-of-service/#3-spamming-and-inappropriate-use-of-github

**Comment by suissa** [comment]
Sobre o 2 eu concordo que o fiz e agora já sei que não posso mais

> You may not advertise in other Users' repositories, such as by posting monetized or excessive bulk content in issues.

Porém o 3 foi culpa dos haters do Be MEAN. Ainda bem que criei outro já sem monetizar.

Obrigado @TiagoDanin
